emojione/index.d.ts

```diff
@@ -0,0 +1,20 @@
+// Type definitions for emojione
+// Project: https://github.com/Ranks/emojione
+// Definitions by: Danilo Bargen <https://github.com/dbrgn/>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+declare var emojione: {
```

Comment:
The type declaration isn't tied to the NPM module, in the sense that installing `@types/emojione` won't install `emojione` along with it.